src/features/Apicalls.js

```diff
@@ -1,15 +1,13 @@
-// import React from 'react';
-// import { useDispatch } from "react-redux";
-// import { setRovers } from "../app/rootSlice";
-
-// const dispatch = useDispatch()
-
 const fetchRovers = async () => {
   const response = await fetch('https://api.nasa.gov/mars-photos/api/v1/rovers/?api_key=vnSCs1JJ7ERtXeAqN6cajKwEh99pz5q6xueRhxMV')
   const rovers = await response.json()
-  console.log('rovers', rovers)
-//   dispatch(setRovers(rovers.rovers))
   return rovers
 }
 
-export default fetchRovers;
\ No newline at end of file
+const fetchImages = async () => {
+  const response = await fetch('https://api.nasa.gov/mars-photos/api/v1/rovers/opportunity/photos?sol=1&api_key=vnSCs1JJ7ERtXeAqN6cajKwEh99pz5q6xueRhxMV')
+  const photos = await response.json()
+  return photos
+}
```

Comment:
Consider how you might include code to handle if you get a bad response from the network request.